Dockerfile

```diff
@@ -24,44 +19,18 @@ COPY . .
 
 RUN CGO_ENABLED=0 make bins
 
-# Download dockerize
-FROM alpine:3.12 AS dockerize
-
-ENV DOCKERIZE_VERSION v0.6.1
-RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-    && tar -C /usr/local/bin -xzvf dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-    && rm dockerize-alpine-linux-amd64-$DOCKERIZE_VERSION.tar.gz \
-    && echo "**** fix for host id mapping error ****" \
-    && chown root:root /usr/local/bin/dockerize
-
-# Alpine base image
-FROM alpine:3.12 AS alpine
-
-RUN apk add --update --no-cache \
-    ca-certificates \
-    tzdata \
-    bash \
-    curl \
-    vim
-
-# set up nsswitch.conf for Go's "netgo" implementation
-# https://github.com/gliderlabs/docker-alpine/issues/367#issuecomment-424546457
-RUN test ! -e /etc/nsswitch.conf && echo 'hosts: files dns' > /etc/nsswitch.conf
-
-SHELL ["/bin/bash", "-c"]
-
 # Temporal server
-FROM alpine AS temporal-server
+FROM temporalio/base-server AS temporal-server
 
 ENV TEMPORAL_HOME /etc/temporal
 RUN mkdir -p /etc/temporal
 
-COPY --from=dockerize /usr/local/bin/dockerize /usr/local/bin
-COPY --from=builder /temporal/temporal-cassandra-tool /usr/local/bin
-COPY --from=builder /temporal/temporal-sql-tool /usr/local/bin
-COPY --from=builder /temporal/tctl /usr/local/bin
-COPY --from=builder /temporal/temporal-server /usr/local/bin
-COPY --from=builder /temporal/schema /etc/temporal/schema
+COPY --from=temporalio/base-server /usr/local/bin/dockerize /usr/local/bin
```

Comment:
This will copy from `temporalio/base-server` to `temporalio/base-server`. I think it can be removed.

Reply:
looks like it, removed